### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by SZRabinowitz** [REAndroid/APKEditor#78] [Ideas]
This is my offer to make a website with an in depth documentation or this APKEditor project. Let me know if you want this @REAndroid. 

In this way, you can keep coding without worrying about the documentation, I will take care of it.

**Comment by REAndroid** [comment]
You are very welcomed, 2FA is restricting my access, I might not be active on discussions/issues but i will do any PR.

**Reply by SZRabinowitz** [reply]
Wow, I am sorry to hear that! 

Would you need any help setting up 2FA? 

On windows computers you can use [2fast](https://github.com/2fast-team/2fast), or on android you can use apps like microsoft authenticator etc. 

Do you have something against 2FA? If so, maybe we can maintain this project outside of this platform. I have some ideas for you, if you are interested I would love if you can send me an email biden2020prez@gmail.com. I would be sad to see this project dissappear. Thanks for all the work you did until now on this amazing tool!!!

**Reply by REAndroid** [reply]
Am back fully, let me know if you are still active
